src/gap.c

```diff
@@ -876,6 +876,36 @@ Obj FuncRUNTIMES( Obj     self)
 }
 
 
+
+/****************************************************************************
+**
+*F  FuncMicrosecondsSinceEpoch( <self> )
+**
+**  'FuncMicrosecondsSinceEpoch' returns the current time since 1.1.1970 in
+**  microseconds.
+**
+**  The accuracy of this is system dependent.
+**
+**  Note that gettimeofday has been marked obsolete in the POSIX standard,
```

Comment:
Perhaps change `,` to `.`, then start new sentence? (It's quite German to have these looong sentences ;-)